File: app/src/main/java/i/am/shiro/amai/Migration.java
package i.am.shiro.amai;

import io.realm.DynamicRealm;
import io.realm.RealmMigration;
import io.realm.RealmSchema;

final class Migration implements RealmMigration {

    @Override
    public void migrate(DynamicRealm realm, long oldVersion, long newVersion) {
        RealmSchema schema = realm.getSchema();
        if (oldVersion == 0) {
            schema.get("SearchModel")
                .removeField("isLoading");
            oldVersion++;
        }
    }
}

File: app/src/main/java/i/am/shiro/amai/Migration.kt
package i.am.shiro.amai

import io.realm.DynamicRealm
import io.realm.RealmMigration

internal class Migration : RealmMigration {
    override fun migrate(realm: DynamicRealm, oldVersion: Long, newVersion: Long) {
        if (oldVersion == 0L) {
            realm.schema["SearchModel"]!!.removeField("isLoading")
        }
    }
}